meerk40t/kernel.py

```diff
@@ -2647,9 +2646,11 @@ def __repr__(self):
             repr(self.line_end),
         )
 
-    def __call__(self, message: Union[str, bytes, bytearray], *args, **kwargs):
+    def __call__(self, message: Union[str, bytes, bytearray], *args, indent=True, **kwargs):
         if self.line_end is not None:
             message = message + self.line_end
+        if indent and not isinstance(message, (bytes, bytearray)):
+            message = "    " + message.replace("\n", "\n    ")
```

Comment:
It seems like indent should maybe be part of the channel values. But, I guess you could want indenting and non-indenting.

Reply:
The idea here is to assist the user in separating commands from output - I experimented with various solutions before deciding to go with indentation. I am very open to alternative suggestions.

Default text is indented and input commands are unindented from that.

Reply:
In actual console, commands have the > prompt in front of them. That makes more sense to me, to replicate in the console window (i.e. any command that is run and appears in the console window without you having typed it gets a > in front of it.)